std/encoding/csv.ts

```diff
@@ -343,10 +343,16 @@ export interface HeaderOptions {
 
 export interface ParseOptions extends ReadOptions {
   /**
-   * If a boolean is provided, the first line will be used as Header definitions.
-   * If `string[]` or `HeaderOptions[]` those names will be used for header definition.
+   * If you provide `header: true` and `columns`, the first line will be skipped.
+   * If you provide `header: true` but not `columns`, the first line will be skipped and used as header definitions.
    */
-  header: boolean | string[] | HeaderOptions[];
+  header?: boolean;
```

Comment:
@jsejcksn Is this correct?
```suggestion
  skipFirstRow?: boolean;
```

Reply:
LGTM